### DISCUSSION TITLE
Browser should forward all keys to Teleport

### DISCUSSION THREAD

**Discussion by bero** [gravitational/teleport#40074] [Q&A] [answered]
I am a member in a team that have Windows in GCP (Google Cloud Platform) and I use Teleport.
I use Ubunt0 23.10 as OS and KVM 8.0.4 as virtualization.
GCP is Windows Server 2022 Datacenter version 21H2.

Generally it works well but there are some problems. To get discussion more focused I only write one problem in each thread.

Some keys like Ctrl+F4 quit browser instead of application inside the tab. In a way this make sense from browsers point of view. But ideally browser should forward all keys to Teleport page when it is active. Maybe some browser extension can do this ?

**Comment by webvictim** [comment, marked as answer]
I've converted this discussion to a feature request: https://github.com/gravitational/teleport/issues/40205

Thank you!
